.changeset/silver-jars-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudflare/next-on-pages': patch
+---
+
+Pin `next` and `vercel` dependencies
```

Comment:
Maybe also mention that the package is being deprecated so we pin version to the latest working version of `next` and `vercel`?

IMO it would be nice to see the deprecation in the changelog